examples/config-keystone.yaml

```diff
@@ -0,0 +1,55 @@
+# The base path of dex and the external name of the OpenID Connect service.
```

Comment:
I don't think we need one of these example files for every connector. Let's remove it. 😃